hw1/02-palindrome.js

```diff
@@ -1,3 +1,41 @@
-const elem = document.querySelector('input');
+// Declarations
+const elem = document.querySelector("input");
+elem.addEventListener("change", handleInput); // get input on enter
+const output = document.getElementById("result"); // output the result to HTML
 
-elem.addEventListener('input', handleInput);
+// Check for positive number
+function isPositiveNumber(num) {
+  if (num < 0) {
+    return false;
+  }
+}
+
+// Reverse a number
+function reverseNumber(num) {
+  let result = "";
```

Comment:
You can make this const if you want, no harm in the let, but the result is being modified each loop in your for so you're just overwriting data. Remember, strings aren't primitive so you can add to them, but you can't modify a specific element of them.

Reply:
I didn't realize strings weren't primitive.  Thanks for the heads up!